### ISSUE TITLE
Default color palette fails WCAG 2 contrast guidelines

### ISSUE THREAD

**Issue by cvrebert** [kurisubrooks/caramel#15]
FWICT using https://addons.mozilla.org/en-us/firefox/addon/wcag-contrast-checker/ , it seems that several of Caramel's default background colors for its widgets don't have a sufficient contrast ratio against their white inner text, thus making them fail [WCAG 2's level AA accessibility guidelines regarding sufficient color contrast for text.](http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
Would you consider adjusting the color palette to address this, or else at least mentioning this caveat in the docs?

**Comment by kurisubrooks** [comment]
default background colours for widgets, meaning the colours of.. everything?

**Comment by cvrebert** [comment]
Mostly. Default and Dark seemed fine and I didn't check the Notices, but yeah, at least the Buttons, Panels, and Labels are affected.

**Comment by kurisubrooks** [comment]
So the tone of the white between the tone of the background colours?

**Comment by cvrebert** [comment]
The contrast ratio, yeah.

**Comment by kurisubrooks** [comment]
Darn that sucks. It took me a while to cherry pick those colours :(
Also, how do I use the tool you linked me?

**Comment by cvrebert** [comment]
Tools=>WCAG Contrast Checker
Select "WCAG 2" tab, and "Level AA" radio button near the bottom.
Check marks are good, Xs are bad.
Selecting a row gives the relevant elements on the page a dashed red border.

**Comment by kurisubrooks** [comment]
According to this tool, Error, Success, Info, Warn are all non-compliant? What is WCAG, could you do some clarification for me? Do we need it?

If we're going to have to use it, all the colours are going to have to look ALOT worse. As I said before, it took us a very long time to pick out the perfect colours and shades. :(

**Comment by kurisubrooks** [comment]
I just checked Bootstrap with this tool, and even their colours aren't compliant. So I think we're going to drop this for now. If it's a real problem, reply in a comment, but for now i'm going to close this issue.
